README.md

```diff
@@ -44,9 +46,13 @@ behavior to code-of-conduct@atomist.com.
 ## Documentation
 
 Please see [docs.atomist.com][atomist-doc] for
-[developer][atomist-doc-sdm] documentation.
+[developer][atomist-doc-sdm] documentation and for high-level documentation of [this pack][atomist-doc-k8-pack].
+
+Also see the TypeScript [API Doc][apidoc] for this pack.
 
 [atomist-doc-sdm]: https://docs.atomist.com/developer/sdm/ (Atomist Documentation - SDM Developer)
+[atomist-doc-k8-pack]: https://docs.atomist.com/pack/k8s/ (Atomist K8s Pack Documentation)
```

Comment:
This link seems to be https://docs.atomist.com/pack/kubernetes/

Reply:
thanks! I was going to check that. Main motivation for the PR was to check the build.